stackstorm-nexus3/README.md

```diff
@@ -0,0 +1,80 @@
+# Stackstorm-Sontatype Nexus3
+
+
+Sontatype nexus3 stackstorm pack
+
+## Installation
+
+
+Install this pack with: `st2 pack install file:///$PWD`
```

Comment:
One too many slashes here - this will result in `file:////present/working/directory` when you just need `file:///present/working/directory`.

Reply:
I copied it from the office docs: https://docs.stackstorm.com/reference/packs.html . So, how should i be writing it?  ``st2 pack install file://$PWD`

Reply:
Yes, `st2 pack install file://$PWD` is the way you should do it.

Reply:
I've fixed that from all over.  sorry to make you repeat